development/commit-messages.md

```diff
@@ -0,0 +1,29 @@
+# Commit messages
+
+Write the summary line and description of what you have done in the imperative
+mode, that is as if you were commanding someone. For example, start the line
+with "Fix", "Add" or "Change" instead of "Fixed", "Added" or "Changed".
+
+Limit your commit summary (the first line of the commit) to 50 characters. If
+you need to provide more detail (which you often will), leave a blank line and
+add a commit description.
+
+You should make sure you capture the context and reasoning for the change:
+
+- Why is the change necessary?
+- How does the commit address the issue?
+- What are the consequences of the change?
+
+You should link to any supporting information:
+
+- relevant guidance, third party documentation or blog posts that informed
+  your decision
+- Trello cards, GitHub issues or other bug reports
+
+You should also refer to the [service manual guidance on commit
+messages](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding#writing-commit-messages).
+
+### Further reading
+- [The art of the commit – David Demaree, A List Apart (February 2016)](https://alistapart.com/article/the-art-of-the-commit)
+- [Writing good commit messages - Erlang wiki](https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
+- [How To Write A Proper Git Commit Message – Stephen Amaza (March 2017)](https://medium.com/@steveamaza/how-to-write-a-proper-git-commit-message-e028865e5791)
```

Comment:
- [What is with the 50/72 rule? - Preslav Rachev, Medium (February 2015)](https://medium.com/@preslavrachev/what-s-with-the-50-72-rule-8a906f61f09c)